src/locales/en/common.json

```diff
@@ -48,15 +53,21 @@
   "LSK received": "LSK received",
   "Language": "Language",
   "Last 24 hours": "Last 24 hours",
-  "Last {{count}} days": "Last {{count}} day",
-  "Last {{count}} days_plural": "Last {{count}} days",
+  "Last {{count}} days": "Last {{count}} days",
+  "Last {{count}} days_plural": "",
```

Comment:
These two lines should not be changed. It's my bad. I've sent you wrong common.json.